src/stories/Library/typography/typography.scss

```diff
@@ -1,4 +1,64 @@
-@import "https://fonts.googleapis.com/css2?family=Gochi+Hand&family=Lora&family=Noto+Sans+JP:wght@400;500&display=swap";
+/* noto-sans-jp-regular - latin */
```

Comment:
Consider adding a comment explaining the source of these font declarations. This will probably help in the future  if we need to add additional weights, styles or even new fonts.

The current font file names also seem infer version names (v42, v26 etc.) indicating that these are updated. Going forward we will need to update these ourselves and being aware of that tool to use here is also helpful.

Reply:
Great idea!